### ISSUE TITLE
Coverall does not take into account extension methods

### ISSUE THREAD

**Issue by jecisc** [hpi-swa/smalltalkCI#424]
I have a project with many extensions methods but the coverage seems wrong because I have the impression they are not taken into account. 

It would be nice to cover them.

**Comment by fniephaus** [comment]
Yes, you're right. [`SCICodeCoverage`](https://github.com/hpi-swa/smalltalkCI/tree/master/repository/SmalltalkCI-Coverage-Core.package/SCICodeCoverage.class) needs to be extended accordingly, contributions are welcome! :)
